src/lib.rs

```diff
@@ -348,6 +348,304 @@ languages_table!(
     (Zu, "zu", "zul", "zul", "Zulu", "Niger–Congo"),
 );
 
+#[cfg(feature = "iterable")]
+pub mod iter {
```

Comment:
I don't think these should be put in their own module, it would probably be fine to have them in the crate root.

To avoid putting `#[cfg]` on everything, you can have this be a private module and then do `#[cfg(feature = "iterable")] pub use iter::*;`. Or alternatively, you don't have to feature-flag this at all, since there aren't dependencies and it's only a bit of code.

Reply:
Well I kept the feature so far (to avoid compiling a 184-long and 26-long arrays when unnecessary).
I moved to a separate private module for clarity.

Reply:
As a side note I think when it will be ready and everything ok, the commits should be squashed into a single commit for clarity.